src/main/java/de/maxhenkel/pipez/events/ModelEvents.java

```diff
@@ -0,0 +1,29 @@
+package de.maxhenkel.pipez.events;
+
+import de.maxhenkel.pipez.Main;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.ModelRegistryEvent;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(modid = Main.MODID, bus = Mod.EventBusSubscriber.Bus.MOD, value = Dist.CLIENT)
```

Comment:
Please register the event in the Main class for consistency.

Reply:
Done.